src/app/components/App.scss

```diff
@@ -173,7 +173,7 @@
         }
     }
 }
-.downvoted {
+.downvoted .downvoted {
```

Comment:
As I've added .downvoted to the .Comment container it will also change the opacity of the children comments. This targets just the downvoted comment not the children.

Reply:
I'm not sure I understand what the intent is -- is this fixing a bug or changing behavior?

Reply:
When a comment is downvoted, the DOM looks like this:
![Screen Shot 2019-06-06 at 9 46 03 am](https://user-images.githubusercontent.com/310654/58997421-ec487700-883f-11e9-8fca-42e309d0fe17.jpg)

The .Comment div has a child div with a class .downvoted which sets its opacity to 0.5.

In order to fix the original issue for this PR, I had to add the .downvoted class to the .Comment div but by doing so, the 0.5 opacity was also applied to all nested comments under it. So making `.downvoted .downvoted` allowed me to target just the first comment, not its children (replies).

Reply:
But I tested again, although it works in most situation, it does not fix the original issue when we have multiple nested downvoted comments.

Will see, if there is another way...